src/stars.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-size: contain;
+  margin: 1px;
```

Comment:
Look at the mockup - margin between stars should be 4px, but pay attention, that for last stars should be no margins.
I think it will be look like 
.stras__star {
  margim-right: ...;
} 
.stars__star:last-child {
  margin-right: 0;
}

Maybe it will be useful for you